src/codeu/chat/client/core/View.java

```diff
@@ -42,11 +42,17 @@
   private final static Logger.Log LOG = Logger.newLog(View.class);
 
   private final ConnectionSource source;
+  private User user;
 
   public View(ConnectionSource source) {
     this.source = source;
   }
```

Comment:
ok. if this works then it's fine for now, but high-level note here is that generally NullPointerExceptions indicate a bug in the code. clean solution here could be to create a new UserView which is composed of a View plus a User, which is used in contexts where a user is signed in.